### DISCUSSION TITLE
求助，咨询一个插件

### DISCUSSION THREAD

**Discussion by mjsoftking** [jxxghp/MoviePilot#2830] [问题咨询]
可以自动开始TR中辅种成功的种子，进行自动做种的插件。请求哪位大佬告知一下插件名称及仓库地址，原本就有这个插件，升级NAS导致重装后找不到了

**Comment by cng10001** [comment]
https://github.com/hotlcc/MoviePilot-Plugins-Third/
下载器助手吗

**Reply by mjsoftking** [reply]
就是这个，感谢
